app/assets/stylesheets/recordings.scss

```diff
@@ -1,11 +1,12 @@
-// Originally added for recordings table
-th{
-  font-weight: normal;
-}
 
-.recordings {
-  &:hover {
-    cursor: pointer;
-    background: rgba(26, 69, 74, 0.2)
+.recordings-list {
+  th {
+    font-weight: normal;
+  }
+
+  .recording-icon-circle {
+    width: 40px;
+    height: 40px;
+    background-color: $primary-light;
```

Comment:
Can also be done in bootstrap

Reply:
Doesn't seem to be working.
I could add a bg-primary-light class if you prefer.